packages/runtime/src/api-handler/preview.ts

```diff
@@ -0,0 +1,5 @@
+export const MAKESWIFT_SITE_VERSION_COOKIE = 'makeswift-site-version'
+
+export const SearchParams = {
+  PreviewToken: 'makeswift-preview-token',
+} as const
```

Comment:
Moved here from the [Next-specific implementation](https://github.com/makeswift/makeswift/pull/1106/files#diff-f0d260034da049c2c8d11d3185d6a2d6918543739374220708d3a1498a0cbf7dL4); other framework implementations should default to the same search param/cookie names.